spec/components/bali/side_menu_spec.rb

```diff
@@ -54,4 +54,14 @@
 
     expect(rendered_component).to have_css 'a.is-active', text: 'item'
   end
+
+  it 'renders a disabled link' do
```

Comment:
How about adding this case in the side_menu preview too?

Reply:
i'm down for it.